classes/class3/README.md

```diff
@@ -1 +1,364 @@
+
+First off, we will be using [Handlebars](http://handlebarsjs.com/) as a templating engine. The default templating engine of Express is actually [Jade](http://jade-lang.com/). We're going to use Handlebars for a few reasons:
+* It more closely resembles the HTML result you will see on the client. Some people consider this a negative, but we like it and it can make debugging your views *much* easier. It also makes it easier while you are learning HTML.
+* It supports pre-compiled templates and client side rendering. We won't get to these for a while, but they provide some great performance boosts and some awesome flexibility when it comes to designing and building dynamic front-ends.
+* Jade's whitespace dependency and style can be a bit too dense and harder to understand. Handlebars may be more "cluttered", but it can also be very nicely structured and layed out in an easy to read manner.
+
+At the end of the day, templating engines-like most things-come down to personal preference, and that's why we're forcing you to like Handlebars! (At least for the next couple weeks).
+
+### Making Express Render Handlebars
+
+To get Express to render handlebars, as well as to add some useful templating features which we will get into later, we are going to use the [Express-Handlebars](https://github.com/ericf/express-handlebars) module. So go ahead and `npm --save install express-handlebars` to get it installed.
+
+Now two more things to know about are layouts and partials.
+* Layouts: Basically meta-templates. They are handlebars documents that allow you to create a wrapper around all the templates that use them.
+* Partials: Basically mini-reusable templates. They are handlebars documents that can be written once and "included" within other templates. (ei. a navigation bar that appears the same on every page)
+
+We're going to build a layout and leave utilizing partials as an exercise for later.
+
+To render a template the first thing we want is a base layout. To start we'll make a very simple one:
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Express Intro</title>
+  </head>
+  <body>
+    {{{body}}}
+  </body>
+</html>
+```
+This is some of the boiler plate for HTML with the center bit `{{{body}}}`. That signifies where templates will be inserted into this layout.
```

Comment:
boiler plate -> [boilerplate](http://en.wikipedia.org/wiki/Boilerplate_code)
Actually include the link, it's a term most probably haven't encountered before.

with the center... signifies -> where `{{{body}}}` signifies